query/src/main/java/org/infinispan/query/impl/InfinispanQueryStatisticsInfo.java

```diff
@@ -17,13 +17,6 @@
 public class InfinispanQueryStatisticsInfo implements InfinispanQueryStatisticsInfoMBean {
 
```

Comment:
This class and InfinispanQueryStatisticsInfoMBean were introduced as a workaround for HSEARCH-1461. I believe they need to be removed.